charts/victoria-metrics-agent/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+weight: 2
+title: Victoria Metrics Agent
```

Comment:
"Victoria Metrics" should be "VictoriaMetrics", would you mind fix them all in charts?

Reply:
done